app/controllers/pupils/schools_controller.rb

```diff
@@ -0,0 +1,2 @@
+class Pupils::SchoolsController < SchoolsController
```

Comment:
I think this pattern is OK for now but I suspect we'll come across a a situation where we might want to inherit from a common `PupilsController` at some point and we'll have to decide what to do.

Reply:
Yep, fair enough